tests/music-learning-contract.test.mjs

```
@@ -0,0 +1,99 @@
+    "/music/learn/guitar",
+    "/music/learn/production",
+    "/music/learn/orchestration",
+    "/music/learn/science",
+  ]) {
+    assert.match(hub, new RegExp(`href: ["']${route}["']`));
+  }
+});
+
+test("the hub is a free editorial layer with explicit privacy, guardian, and rights boundaries", () => {
+  const source = compact(hub);
+  assert.match(source, /free, self-paced map/i);
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)